static/app/views/settings/organizationGeneralSettings/index.tsx

```diff
@@ -77,6 +78,13 @@ function OrganizationGeneralSettings(props: Props) {
     } else {
       // This will update OrganizationStore (as well as OrganizationsStore
       // which is slightly incorrect because it has summaries vs a detailed org)
+      if (prevData.codecovAccess !== updated.codecovAccess) {
```

Comment:
Should this code be moved above the comment?

Reply:
Missed this comment, good call. I'll fix it in another PR.